configs/mobile/README_en.md

```diff
@@ -48,13 +53,27 @@ This directory contains models optimized for mobile applications, at present the
     |                  | SD625   | SD835   | SD845   | SD855   | Kirin 970 | Kirin 980 |
     |------------------|---------|---------|---------|---------|-----------|-----------|
     | SSDLite Large    | 107.535 | 51.1382 | 34.6392 | 20.4978 | 50.5598   | 24.5318   |
+    | SSDLite Large Quant |  |  |   |  |    |    |
     | SSDLite Small    | 51.5704 | 24.5156 | 18.5486 | 11.4218 | 24.9946   | 16.7158   |
+    | SSDLite Small Quant |  |  |  | |   |    |
     | YOLOv3 baseline  | 413.486 | 184.248 | 133.624 | 75.7354 | 202.263   | 126.435   |
     | YOLOv3 prune     | 98.5472 | 53.6228 | 34.4306 | 21.3112 | 44.0722   | 31.201    |
     | Cascade RCNN 320 | 131.515 | 59.6026 | 39.4338 | 23.5802 | 58.5046   | 36.9486   |
     | Cascade RCNN 640 | 473.083 | 224.543 | 156.205 | 100.686 | 231.108   | 138.391   |
 
 
+## Notes on SSDLite quantization
+
+In the SSDLite model, we use a complete quantitative training method to train the model. A total of 400,000 rounds are trained under the 8-card GPU. During the training, `res_conv1` and `se_block` are fixed and not trained. The execution instructions are:
```

Comment:
We use a complete quantitative training method to train the SSDLite model. It is trained for a total of 400,000 rounds with the 8-card GPU. We freeze `res_conv1` and `se_block` . The command used is listed bellow:

Reply:
Done，thx~